src/figuresClasses.ts

```diff
@@ -2,10 +1,74 @@
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('A sides must be greater than zero.');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('The provided sides do not form a triangle.');
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Math.round(area * 100) / 100;
+  }
+}
 
-export class Triangle implements Figure {}
+export class Circle implements Figure {
+  shape: string = 'circle';
 
-export class Circle implements Figure {}
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public radius: number,
+  ) {
+    if (radius <= 0) {
+      throw new Error('Radius must be greater than zero.');
+    }
+  }
 
-export class Rectangle implements Figure {}
+  getArea(): number {
+    const area = Math.PI * this.radius ** 2;
+
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Rectangle implements Figure {
+  shape: string = 'rectangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public height: number,
+    public width: number,
+  ) {
+    if (height <= 0 || width <= 0) {
+      throw new Error('Width and height must be greater than zero.');
+    }
```

Comment:
[CODE STYLE] If you throw an error try to provide its detailed description in error message. For example, instead of 'Width and height must be greater than zero.', you could write 'The width and height of a rectangle must be greater than zero.' to make it more clear.